cdap-app-fabric/src/main/java/io/cdap/cdap/internal/sysapp/SystemAppEnableExecutor.java

```diff
@@ -84,7 +84,7 @@ public void deployAppAndStartPrograms(Arguments arguments) {
         startProgram(program.getProgramId());
       } catch (Exception ex) {
         LOG.error("Failed to start program {} for app {} with exception {}",
-            program.getProgramId(), arguments.getId(), ex);
```

Comment:
style: when there is a newline in the middle of a method call, arguments should line up with the previous line.